### PR TITLE
[feature/APPC-923] - Update transactions empty screen

### PR DESCRIPTION
**Pull request by joaopmgouveia**

### CODE REVIEW

File: app/src/main/java/com/asfoundation/wallet/ui/TransactionsActivity.java [hunk 1]
```diff
@@ -54,9 +52,8 @@
 
   public static final String LEARN_MORE_INFO_URL = "https://appcoins.io/";
```
Comment (Trinkes) [inline app/src/main/java/com/asfoundation/wallet/ui/TransactionsActivity.java]: remove if it's not used anymore

File: app/src/main/res/layout/layout_empty_transactions.xml [hunk 2]
```diff
@@ -1,63 +1,99 @@
...
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      >
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:paddingBottom="55dp"
+        >
+      <TextView
+          android:id="@+id/no_transactions_text"
```
Comment (JoaoRaimundo) [inline app/src/main/res/layout/layout_empty_transactions.xml:19]: Please remove this button and its action. This is no longer needed.

File: app/src/main/java/com/asfoundation/wallet/widget/EmptyTransactionsView.java [hunk 3]
```diff
@@ -6,33 +6,30 @@
...
 public class EmptyTransactionsView extends FrameLayout {
 
-  private final Button airdropButton;
   private final TextView noTransactionsTextView;
+  private final TextView earnBonusTextView;
+  private final LottieAnimationView noTransactionsAnimationView;
 
   public EmptyTransactionsView(@NonNull Context context, OnClickListener onClickListener,
-      boolean isMainNetwork) {
+      boolean isMainNetwork, String bonus) {
```
Comment (JoaoRaimundo) [inline app/src/main/java/com/asfoundation/wallet/widget/EmptyTransactionsView.java]: isMainNetwork value not being used. Please remove it.

File: app/src/main/res/layout/layout_empty_transactions.xml [hunk 4]
```diff
@@ -1,63 +1,84 @@
...
+          app:lottie_loop="true"
+          app:lottie_rawRes="@raw/transactions_empty_screen_animation"
+          />
+
+      <TextView
+          android:id="@+id/earn_bonus_text"
+          android:layout_width="0dp"
+          android:layout_height="wrap_content"
+          android:gravity="center"
+          android:text="@string/gamification_home_body"
```
Comment (JoaoRaimundo) [inline app/src/main/res/layout/layout_empty_transactions.xml]: Adding default string containing replacement tag is not a good if string is not replaced.

File: app/src/main/java/com/asfoundation/wallet/widget/EmptyTransactionsView.java [hunk 5]
```diff
@@ -6,33 +6,30 @@
...
 
     LayoutInflater.from(getContext())
         .inflate(R.layout.layout_empty_transactions, this, true);
 
     noTransactionsTextView = findViewById(R.id.no_transactions_text);
-    airdropButton = findViewById(R.id.action_air_drop);
-    findViewById(R.id.action_learn_more).setOnClickListener(onClickListener);
+    noTransactionsAnimationView = findViewById(R.id.transactions_empty_screen_animation);
+    earnBonusTextView = findViewById(R.id.earn_bonus_text);
+    earnBonusTextView.setText(getResources().getString(R.string.gamification_home_body, bonus));
```
Comment (JoaoRaimundo) [inline app/src/main/java/com/asfoundation/wallet/widget/EmptyTransactionsView.java:27]: Validation for null string is not being done, this may go wrong.

**Review by Trinkes — CHANGES_REQUESTED** [review]

**Review by Trinkes — APPROVED** [review]

**Pull request merged**